docs/opcodes/55/berlin.mdx

```diff
@@ -1,5 +1,10 @@
 ## Gas
 
+### Definitions:
+- **value**: The value of the storage slot after executing the opcode SSTORE.
+- **current_value**: The value of the storage slot before executing the opcode SSTORE.
+- **original_value**: The value of the storage slot at the beginning of the transaction.
+##
```

Comment:
The extra ## should not be there.

Reply:
Changed it to a code section

Reply:
Please revert, the extra indent already makes it a code section :-)